apps/css-workshop/src/components/Button.astro

```diff
@@ -1,22 +1,40 @@
 ---
 type Props = {
-  variant?: 'high-visibility' | 'borderless' | 'cta';
+  variant?: 'high-visibility' | 'borderless' | 'cta' | 'idea';
```

Comment:
Maybe `idea` could be another boolean prop similar to `dropdown`?

Otherwise you'll have to add extra conditional logic to account for the CSS changes in #1919

Reply:
Yeah, that makes sense because it will also be a class. Will change.

Reply:
Added in this [commit](https://github.com/iTwin/iTwinUI/pull/2012/commits/3de64f92c67bfd85550588ad938ada78cb1dc68a).